### PR TITLE
finalization of the integration

### PR DESCRIPTION
**Pull request by RomainLF**

### CODE REVIEW

File: frontend/src/components/Networks/style.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      position: relative;
+    }
+    .cube {
+      position: absolute;
+      width: 35vw;
+      height: 70vh;
+      background-color: var(--yellow);
+      left: 10%;
+      top: 10%;
+      z-index: 1;
```
Comment (FrancoisSalmon) [inline frontend/src/components/Networks/style.jsx:23]: try to do this without z-index

**Review by FrancoisSalmon — CHANGES_REQUESTED** [review]
try to do this without z-index

File: frontend/src/components/Networks/index.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import SNetworks from "./style";
+import imgNetworks from "../../assets/imgNetworks.jpg";
```
Comment (LuigiMenez) [inline frontend/src/components/Networks/index.jsx:2]: Use import absolue

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Use import absolute

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by FrancoisSalmon — APPROVED** [review]
conflicts resolved

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**